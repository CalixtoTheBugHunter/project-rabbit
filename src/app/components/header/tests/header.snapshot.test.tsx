import { render } from '@testing-library/react'
import Header from '../header'
 
it('renders Header unchanged', () => {
  const { container } = render(<Header />)
  expect(container).toMatchSnapshot()
})