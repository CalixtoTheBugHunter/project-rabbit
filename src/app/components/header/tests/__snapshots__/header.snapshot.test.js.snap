// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Header unchanged 1`] = `
<div>
  <h1
    class="h1"
  >
    Hello World
  </h1>
</div>
`;
